number-store-list-table.php

```diff
@@ -121,6 +122,15 @@ public function column_default( $item, $column_name ) {
 					$value = '<strong>' . __('Unknown', 'woocommerce-pdf-ips-number-tools') .'</strong>';
 				}
 				break;
+			case 'date_display' :
```

Comment:
This is a remark, not related with this PR, for those interested: When you select **No** for the **Display invoice date** option:

![image](https://user-images.githubusercontent.com/38109855/237060812-cc377fa4-a5f0-4831-8146-ebd1e0cc4c64.png)

The invoice date is selected by default:

![image](https://user-images.githubusercontent.com/38109855/237060632-9e75a35a-8d03-4d3f-a5f4-ffa8aed31a01.png)